src/pages/Root.tsx

```diff
@@ -48,7 +48,31 @@ export const Root = () => {
 						</Button>
 					</div>
 				</div>
-				<div className="">
+				<Header
+					header="Pet Heart"
+					headerParagraph="s simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has  "
+				/>
+				<div className="bg-backgroundLight pt-5 pb-5">
+					<Title title="Mapy" />
+					<div className="flex items-center space-x-2.5 ml-8 mr-8  ">
```

Comment:
Jak se lisi tohle od komponenty Header? to je jen umisteni obrazku a barva ne? Proto by se to melo predelat do Jedne komponenty, ktera bude mi brat header, children a variant. A About me je v zasade to stejne taky akorat s jinou velikosti obrazku ci blbe koukam? Pokud tomu tak je tak se to jeste rozsiri o imageSize

Reply:
ok uz to vidim a beru zpet. Header je jiny. Ten nechej, ale pro about me a to maps udelej komponentu Section, ktera bude brat to co jsem psal nahore.